juniper/src/schema/meta.rs

```diff
@@ -202,7 +202,7 @@ pub struct Argument<'a, S> {
     #[doc(hidden)]
     pub arg_type: Type<'a>,
     #[doc(hidden)]
-    pub default_value: Option<InputValue<S>>,
+    pub default_value: Option<Spanning<InputValue<S>>>,
```

Comment:
@audunhalland I'm not sure whether this should be `Spanning`, because, judging by code, it always has `Spanning::unlocated()` value. Keeping `Spanning` as a part of type here, may give assumptions that the field is actually `Span`ned here properly, in a way.

Better remove it, and do the `Spanning::unlocated()` wrapping explicitly in the places where this is required.